src/funding/paylater/config.jsx

```diff
@@ -49,6 +56,19 @@ function getLabelText(fundingEligibility : FundingEligibilityType) : ?string {
         labelText = '4X PayPal';
     }
 
+    if(getLocale().includes("US")) {
```

Comment:
please let me know if there is a cleaner way to check that this instance of the buttons are rendering in a US transaction

Reply:
Locale doesn't resolve to the true buyer country, so this can't be used as a reliable check